src/neynar-api/v2/client.ts

```diff
@@ -1065,6 +1065,46 @@ export class NeynarV2APIClient {
     return response.data;
   }
 
+  /**
+   * Retrieves users by their location, specified by latitude and longitude coordinates.
+   * 
+   * @param {number} latitude - The latitude coordinate of the location.
+   * @param {number} longitude - The longitude coordinate of the location.
+   * @param {Object} [options] - Optional parameters to tailor the request.
+   * @param {number} [options.viewerFid] - The FID of the user viewing this information, used for providing contextual data specific to the viewer.
+   * @param {number} [options.limit] - The number of users to fetch per request. Defaults to 25 with a maximum of 100.
+   * @param {string} [options.cursor] - Pagination cursor for fetching specific subsets of results.
+   * 
+   * @returns {Promise<UsersResponse>} A promise that resolves to a `UsersResponse` object, 
+   *   containing information about the users at the specified location.
+   * 
+   * @example
+   * // Example: Fetch users by location with viewer fid
+   * client.fetchUsersByLocation(37.7749, -122.4194, {viewerFid: 3}).then(response => {
+   *  console.log('Users by Location:', response);
+   * });
+   * 
+   * For more information, refer to the [Neynar documentation](https://docs.neynar.com/reference/user-by-location).
+   */
+  public async fetchUsersByLocation(
+    latitude: number,
+    longitude: number,
+    options?: {
+      viewerFid?: number;
+      limit?: number;
+      cursor?: string;
+    }
+  ): Promise<UsersResponse> {
+    const response = await this.apis.user.userByLocation(
```

Comment:
Please update the `operationId` in OAS to `fetch-users-by-location`.
This is because in SDK v2 we'll `operationId` will determine the name of the method attached to the class.